docker_scripts/squash.py

```diff
@@ -518,8 +520,8 @@ def run(self):
             # without loading into Docker
             self._tar_image(self.output_path, new_image_dir)
             self.log.info("Image available at '%s'" % self.output_path)
-        else:
-            # Load image into Docker
+        if not self.output_path or self.load_output_back:
```

Comment:
I think you're thinking about something different :)
What I'm trying to do is produce both tarfile *and* engine image. If I use if/elif/else, I will always be able to go through just one clause, but I need to go through two if `load_output_back` is `True` and `output_path` is not `None`. And then the warning should be emitted assuming neither one of the clauses was gone through. (As I demonstrated in the pseudocode segment in my previous comment).